docs/ref/templates/builtins.txt

```diff
@@ -1849,6 +1849,27 @@ For example:
 If ``value`` is ``"testing\r\njavascript 'string\" <b>escaping</b>"``,
 the output will be ``"testing\\u000D\\u000Ajavascript \\u0027string\\u0022 \\u003Cb\\u003Eescaping\\u003C/b\\u003E"``.
 
+.. templatefilter:: escapeseq
+
+``escapeseq``
+-------------
+
+.. versionadded:: 5.0
+
+Applies the :tfilter:`escape` filter to each element of a sequence. Useful in
+conjunction with other filters that operate on sequences, such as
+:tfilter:`join`. For example:
+
+.. code-block:: html+django
+
+    {% autoescape off %}
+        {{ some_list|escapeseq|join:", " }}
+    {% endautoescape %}
+
+You couldn't use the :tfilter:`escape` filter directly in this case, as it would
+first convert the variable into a string, rather than working with the
+individual elements of the sequence.
```

Comment:
```suggestion
```

Personally I wouldn't put this here, rather under escape with something like "To escape sequences use <link-to-escapeseq>"